heft-plugins/heft-jest-plugin/src/HeftJestReporter.ts

```diff
@@ -37,7 +38,7 @@ export default class HeftJestReporter implements Reporter {
   private _debugMode: boolean;
 
   public constructor(jestConfig: Config.GlobalConfig, options: IHeftJestReporterOptions) {
-    this._terminal = options.heftConfiguration.globalTerminal;
+    this._terminal = options.logger.terminal;
```

Comment:
Huh. Funny that it was doing this before.

Reply:
Yeah seemed a little odd! It's nice to have it all under a scope now, makes it clear where its coming from.